fcrepo-http-api/src/test/java/org/fcrepo/http/api/FedoraLdpTest.java

```diff
@@ -787,9 +789,32 @@ public void testCreateNewBinary() throws RepositoryException, MalformedRdfExcept
         }
     }
 
+    @Test(expected = InsufficientStorageException.class)
+    public void testCreateNewBinaryWithInsufficientResources() throws RepositoryException, MalformedRdfException,
```

Comment:
Minor: spreading the `throws` classes over three lines seems unnecessary. Will then all fit on line `794`?